pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2022 Contributors to the Eclipse Foundation.
+    Copyright (c) 2022-2024 Contributors to the Eclipse Foundation.
```

Comment:
Tiny nit, but we typically use a comma here.

Reply:
Updated in all files :)